pages/common/tldr.md

```diff
@@ -5,3 +5,19 @@
 - Get typical usages of a command (hint: this is how you got here!):
 
 `tldr {{command}}`
+
+- List available pages:
+
+`tldr --list`
+
+- Show the tar tldr page for linux:
+
+`tldr --platform {{linux}} {{tar}}`
+
+- Get help for a command in Chinese:
```

Comment:
Could we remove this example please? `-t` isn't in the spec.

Reply:
You probably meant the following example, not this one.